### PR TITLE
Add  ultra-api project

### PR DESCRIPTION
**Pull request by bambadiagne**
This is a optimized RESTful API. It's built with Flask,PostgresSQL,Redis and Docker.The aim is to show the different concepts of a backend(caching,security,...) that are sometimes considered as just CRUD operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,8 @@ If you are the maintainer of a project, you can add our badge by copying the cod
 
 ## <a name="U"> </a>U
 
+- **[Ultra-Api]([https://github.com/daoodaba975/taaru](https://github.com/bambadiagne/ultra-api))** - This is a optimized RESTful API. It's built with Flask,PostgresSQL,Redis and Docker.The aim is to show the different concepts of a backend(caching,security,...). by **[@bambadiagne](https://github.com/bambadiagne)**
```
Comment (daoodaba975) [inline README.md:224]: Duplicated project link !
need to remove `[https://github.com/daoodaba975/taaru]`

**Review by daoodaba975 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daoodaba975) [reply README.md:224]: @bambadiagne

Comment (bambadiagne) [reply README.md]: it's done @daoodaba975

Comment (daoodaba975) [reply README.md]: Great, made some fix also!

**Pull request merged**